swatch-contracts/src/main/resources/application.properties

```diff
@@ -217,6 +217,12 @@ quarkus.rest-client."com.redhat.swatch.clients.rbac.api.resources.AccessApi".tru
 quarkus.rest-client."com.redhat.swatch.clients.rbac.api.resources.AccessApi".trust-store-password=${clowder.endpoints.rbac-service.trust-store-password}
 quarkus.rest-client."com.redhat.swatch.clients.rbac.api.resources.AccessApi".trust-store-type=${clowder.endpoints.rbac-service.trust-store-type}
 
+quarkus.rest-client."com.redhat.swatch.clients.export.api.resources.ExportApi".url=${clowder.private-endpoints.export-service-service.url:http://localhost:10000}
```

Comment:
Do you know why it's `export-service-service` instead of just `export-service`?

Reply:
This is because the quarkus clowder config source utility composes the key using the "{app}-{service}". The app name is "export-service" and the service is "service" as well.